phpBB/composer.json

```diff
@@ -45,6 +44,9 @@
 		"symfony/finder": "~3.4",
 		"symfony/http-foundation": "~3.4",
 		"symfony/http-kernel": "~3.4",
+		"symfony/polyfill-intl-normalizer": "^1.23",
+		"symfony/polyfill-mbstring": "^1.23",
+		"symfony/polyfill-php72": "^1.23",
```

Comment:
Is this just related to PHP 7.2 or greater?

Reply:
Nope, this polyfill just reimplements utf related functions from PHP 7.2 core.